runtime/tests/runtime_integration_tests/test_pauser_privilege_levels.rs

```diff
@@ -453,3 +453,46 @@ fn test_measurement_log_pl_context_threshold() {
             == u32::from(CaliptraError::RUNTIME_PL0_USED_DPE_CONTEXT_THRESHOLD_REACHED)
     });
 }
+
+#[test]
+fn test_pl0_unset_in_header() {
+    let fuses = Fuses::default();
+    let rom = caliptra_builder::rom_for_fw_integration_tests().unwrap();
+    let mut model = caliptra_hw_model::new(BootParams {
+        init_params: InitParams {
+            rom: &rom,
+            security_state: SecurityState::from(fuses.life_cycle as u32),
+            ..Default::default()
+        },
+        fuses,
+        ..Default::default()
+    })
+    .unwrap();
+
+    let mut opts = ImageOptions::default();
+    opts.vendor_config.pl0_pauser = None;
```

Comment:
This test, as is, does not fail when the change to drivers.rs is reverted. This covers the pl0_pauser not set, and locality != pl0_pauser case that I believe other tests here already cover.

There are two cases I think we should cover:
pl0_pauser_flag not set and locality == pl0_pauser. To hit this we would need to use model.set_apb_pauser to set the pauser to 0. Unfortunately the SW model does not implement this function at the moment.
pl0_pauser_flag set and locality != pl0_pauser. For this we would provide Some(0) (any value other than 1)

Reply:
> This test, as is, does not fail when the change to drivers.rs is reverted

Are you sure? I did check that it failed before making the driver change.

> pl0_pauser_flag not set and locality == pl0_pauser. To hit this we would need to use model.set_apb_pauser to set the pauser to 0. Unfortunately the SW model does not implement this function at the moment.
pl0_pauser_flag set and locality != pl0_pauser. For this we would provide Some(0) (any value other than 1)

Good point. I'll add another test for this case.

Reply:
Added test for the second case.

Reply:
Never mind, you're right that it doesn't fail. Looking into this.

Reply:
Alright, the test ended up being very complicated because I have to change a header bit and then re-sign. But it now fails/passes in the correct scenarios.